File: go/1010.go
package main

import "fmt"

func main() {
	var total_to_pay float64 = 0
	for i := 0; i < 2; i++ {
		var id, quantity int
		var price float64
		fmt.Scanln(&id, &quantity, &price)
		total_to_pay += float64(quantity) * price
	}
	fmt.Printf("VALOR TOTAL A PAGAR = %.2f\n", total_to_pay)
}

File: C/src/1010.c
#include <stdio.h>

int main(){
    int quantity, id;
    float price, total_to_pay = 0;
    for(int i = 0; i < 2; i++){
        scanf("%i %i %f",&id,&quantity,&price);
        total_to_pay += quantity*price
    }
    printf("VALOR A PAGAR: R$ %.2f\n",(total_to_pay));
    return 0;
}